cron.sh

```diff
@@ -65,7 +65,12 @@ while true; do
                 branch="${remote#origin/}"
                 git branch --track "$branch" "$remote";
                 git checkout "$branch"
-                git pull
+                # When the branch cannot be pulled because of any conflict, the remote branch is checkedout
+                if ! git pull; then
+                    git checkout master
+                    git branch -D "$branch"
+                    git checkout "$branch"
```

Comment:
Sorry if this  appears twice, Github's review interface is convenient as ever. I think using `git reset --hard "$remote"` in place of `git pull` makes more sense here. You'll avoid pull failure (which should not heppen anyway unless someone force pushed stuff, or there are incompatible local changes) and won't have to go through the checkout, delete, checkout hoops.

Reply:
Thanks for reviewing this. Change applied.